generatebundlefile/README.md

```diff
@@ -41,10 +41,132 @@ generatebundlefile --generate-sample
 
 This will output a sample bundle file to ./output.
 
+
+### Public Promotion to another Account
+
+```sh
+generatebundlefile --private-profile "profile-name" --input data/sample_input.yaml
```

Comment:
This command, and the one below it are exactly the same. I'm guessing that's a typo? 

If not, it might be worth calling out, and explaining how the same command can provide different results.

Reply:
good callout, copy/paste error.